hyperspy/tests/drawing/test_utils.py

```diff
@@ -15,23 +15,15 @@
 # You should have received a copy of the GNU General Public License
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
-import warnings
-import sys
 from unittest.mock import Mock
 
 import matplotlib
-from matplotlib.testing.decorators import cleanup
 import pytest
 
 import hyperspy.drawing.utils as utils
-from hyperspy.misc.test_utils import assert_warns
 
 
-@pytest.mark.skipif(sys.platform == 'darwin',
-                    reason="Plot testing not supported on osx by travis-ci")
-@cleanup
 def test_create_figure():
-    dummy_warning = 'dummy_function have been called after closing windows'
     if matplotlib.get_backend() not in ("GTKAgg", "WXAgg", "TkAgg", "Qt4Agg"):
```

Comment:
Sorry for not explaining myself: I was questioning if we should not let the test run for all backends since all of them must be supported now that you are using the right mpl event for closing the figure.